src/mono/System.Private.CoreLib/src/System/GC.Mono.cs

```diff
@@ -316,5 +316,10 @@ public static TimeSpan GetTotalPauseDuration()
         {
             return TimeSpan.Zero;
         }
+
+        public static System.Collections.Generic.IReadOnlyDictionary<string, object> GetConfigurationVariables()
+        {
+            throw new NotImplementedException();
```

Comment:
It may be better to return empty Dictionary for these cases instead of throwing.